tests/device-gateway_fake.py

```diff
@@ -18,6 +18,9 @@ class Handler(SimpleHTTPRequestHandler):
     AuthPrefix = "/hub-creds/"
     RegistryAuthPrefix = "/token-auth/"
     RegistryPrefix = "/v2/"
+    def do_POST(self):
```

Comment:
I think it makes sense to remove this change from the given commit and add it to this one https://github.com/foundriesio/aktualizr-lite/pull/82/commits/789b83d982095c26a7e09ed0e7fcceaf3f4e1116.

Reply:
ok moving it to its own commit (I dont think we want to hard reset the tree?)